spec/rollbar/notifier_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rollbar'
+require 'rollbar/notifier'
+
+describe Rollbar::Notifier do
+  describe '#scope' do
+    let(:new_scope) do
+      { 'foo' => 'bar' }
+    end
+    let(:new_config) do
+      { 'environment' => 'foo' }
+    end
+
+    it 'creates a new notifier with merged scope and configuration' do
+      new_notifier = subject.scope(new_scope, new_config)
+
+      expect(new_notifier).not_to be(subject)
+      expect(new_notifier.configuration.environment).to be_eql('foo')
```

Comment:
Can you also make sure that the parent notifier's configuration has not been changed?